### PR TITLE
add gtn-downloader blogpost

### PR DESCRIPTION
**Pull request by PlushZ**

### CODE REVIEW

File: content/news/2024-05-27-gtn-downloader/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+## Seamless Integration with OneData
+
+In addition to local downloads, the GTN Downloader can upload data to OneData, a cloud storage solution. This integration ensures that the latest GTN data is always available to users.
+
+### Automated Workflow with GitHub CI/CD:
+- **Automated Workflow**: A GitHub Actions workflow runs once a week on weekends to download the latest data from the GTN tutorials and upload it to OneData.
+- **Environment Setup**: The workflow sets up necessary environment variables and installs dependencies, including Oneclient, the OneData client.
+- **Data Upload**: After downloading the data, the workflow uploads it to OneData, making it accessible via the cloud.
+
+## Accessing GTN Data in the Cloud
```
Comment (bgruening) [inline content/news/2024-05-27-gtn-downloader/index.md:33]: I would move this section up, I guess this is what we want to advertise the most.

File: content/news/2024-05-27-gtn-downloader/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+3. **Galaxy Server Integration**: Access the data on the [European Galaxy server](https://usegalaxy.eu/). Go to the "Upload data" button, select "Choose remote files," and navigate to the GTN repository.
+
+### Configuration Example:
+To access the GTN data via OneData or include it in your Galaxy server, use the following configuration:
+```yaml
+- type: onedata
+  id: gtn_public_onedata
+  label: GTN training data
+  doc: Training data from the Galaxy Training Network (powered by Onedata)
+  accessToken: "MDAxY2xvY2F00aW9uIGRhdGFodWIuZWdpLmV1CjAwNmJpZGVudGlmaWVyIDIvbm1kL3Vzci00yNmI4ZTZiMDlkNDdjNGFkN2E3NTU00YzgzOGE3MjgyY2NoNTNhNS9hY3QvMGJiZmY1NWU4NDRiMWJjZGEwNmFlODViM2JmYmRhNjRjaDU00YjYKMDAxNmNpZCBkYXRhLnJlYWRvbmx5CjAwNDljaWQgZGF00YS5wYXRoID00gTHpaa1pUTTROMkl4WmpjMllXVmpOMlU00WWpreU5XWmtNV00ZpT1RKbU1ETXlZMmhoWTJReAowMDJmc2lnbmF00dXJlIIQvnXp01Oey02LnaNwEkFJAyArzhHN8SlXSYFsBbSkqdqCg"
```
Comment (bgruening) [inline content/news/2024-05-27-gtn-downloader/index.md:48]:
```suggestion
  # The following token is a public, read-only token that can be shared.
  accessToken: "MDAxY2xvY2F00aW9uIGRhdGFodWIuZWdpLmV1CjAwNmJpZGVudGlmaWVyIDIvbm1kL3Vzci00yNmI4ZTZiMDlkNDdjNGFkN2E3NTU00YzgzOGE3MjgyY2NoNTNhNS9hY3QvMGJiZmY1NWU4NDRiMWJjZGEwNmFlODViM2JmYmRhNjRjaDU00YjYKMDAxNmNpZCBkYXRhLnJlYWRvbmx5CjAwNDljaWQgZGF00YS5wYXRoID00gTHpaa1pUTTROMkl4WmpjMllXVmpOMlU00WWpreU5XWmtNV00ZpT1RKbU1ETXlZMmhoWTJReAowMDJmc2lnbmF00dXJlIIQvnXp01Oey02LnaNwEkFJAyArzhHN8SlXSYFsBbSkqdqCg"
```

**Review by bgruening — COMMENTED** [review]
Thanks a lot @PlushZ

File: content/news/2024-05-27-gtn-downloader/index.md [hunk 3]
```diff
@@ -1,24 +1,27 @@
...
-subsites: [global, all]
+tags: [esg-wp4, esg]
+subsites: [global, all, esg]
 main_subsite: eu
 ---
 
-# GTN Downloader: Simplifying Access to Galaxy Training Network Data
-
-[GTN-Downloader](https://github.com/usegalaxy-eu/gtn-downloader) makes it easier for users to access and organize data from the Galaxy Training Network (GTN). Here’s a straightforward overview of what this feature does and how it can benefit you.
+As part of the [EuroScienceGateway](https://eurosciencegateway.eu/) and in cooperation with [OneData](https://onedata.org) and [EGI](https://www.egi.eu/) we are providing all [GTN](https://training.galaxyproject.org/) training data on a public accessible [could storage](https://datahub.egi.eu/share/2697e33bd34f1870b0961414b8c77753chf583).
```
Comment (sebastian-luna-valero) [inline content/news/2024-05-27-gtn-downloader/index.md]:
```suggestion
As part of the [EuroScienceGateway](https://eurosciencegateway.eu/) and in cooperation with [OneData](https://onedata.org) and [EGI](https://www.egi.eu/) we are providing all [GTN](https://training.galaxyproject.org/) training data on a public accessible [cloud storage](https://datahub.egi.eu/share/2697e33bd34f1870b0961414b8c77753chf583). 
```

**Comment by lopiola** [comment]
Hello, I was invited to chip in regarding the Onedata stuff. Here is the PR with my suggestions: https://github.com/PlushZ/galaxy-hub/pull/3

File: content/news/2024-05-27-gtn-downloader/index.md (hunk 3 above)
Comment (bgruening) [reply content/news/2024-05-27-gtn-downloader/index.md]: Thanks!

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks a lot everyone!

**Pull request merged**